styles/main.css

```diff
@@ -862,6 +862,39 @@ button {
 }
 
 /*profile stuff to be moved later*/
+.verify-status-dependent {
+  display: none;
```

Comment:
We should be cautious about throwing in too many `!important`s. Is this justified in this context?